README.md

```diff
@@ -1,4 +1,32 @@
-# codewind-odo-extension
-Extension to Codewind providing support for odo projects: https://codewind.dev
+# Codewind ODO extension
+Extension to Codewind providing support for ODO projects: https://codewind.dev
 
-README.md is in progress and will be completed soon.
+## Add additional rules to support Codewind ODO extension
+ODO extension needs add additional rules for accessing OpenShift resources
+Steps to add additional rules:
```

Comment:
I think we could cut this line. For instance, in Install Codewind ODO extension manually we don't precede the steps with "Steps to install extension manually." We simply go straight into the steps.

So I think we can do that here.